app/configureStore.js

```diff
@@ -37,18 +32,18 @@ export default function configureStore(initialState = {}, history) {
   const middlewares = [sagaMiddleware, routerMiddleware(history)];
 
   const enhancers = [
-    applyMiddleware(...middlewares),
     createInjectorsEnhancer({
       createReducer,
       runSaga,
     }),
   ];
 
-  const store = createStore(
-    createReducer(),
-    initialState,
-    composeEnhancers(...enhancers),
-  );
+  const store = configureStore({
+    reducer: createReducer(),
+    preloadedState: initialState,
```

Comment:
Is there a rationale for the API using `preloadedState` instead of `initialState`? Is this something we should change in `configureAppStore` and other places too?

(Saw your tweet regarding this btw, we have an ongoing discussion for an eventual move to TS.)

Reply:
Naming-wise, I think the earlier Redux docs did refer to the `createStore` signature as `createStore(reducer, initialState)`.  At some point there was a discussion about intended use cases, and the argument was that "this is really meant for loading state that was persisted or something", so we came up with `preloadedState` as the name.  That name is used in the docs:

https://redux.js.org/recipes/structuring-reducers/initializing-state#initializing-state

https://redux.js.org/api/createstore

So, I reused that name with RTK's `configureStore`.